popsycle/synthetic.py

```diff
@@ -693,14 +780,17 @@ def _make_comp_dict(iso_dir, log_age, currentClusterMass, star_dict, next_id,
     comp_dict = None
 
     # Calculate the initial cluster mass
-    massLimits = np.array([0.1, 0.5, 120]) # changed from 0.08 to 0.1 at start because MIST can't handle.
+    massLimits = np.array([0.1, 0.5,
```

Comment:
minor thing: don't break in the code... just move the comment up or down.

Reply:
fixed